frontend/src/components/ManageEventsPage/HomeModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import Modal from "@mui/material/Modal";
```

Comment:
What is this file for? It just looks like a duplicate of the HomeModal from the HomePage folder.

Reply:
Oh yeah, I accidentally left that in haha. We can delete it